handlers/test/unit/handlers/test_rpm.py

```diff
@@ -0,0 +1,352 @@
+from unittest import TestCase
+
+from mock import patch, Mock
+
+from pulp_rpm.handlers.rpm import (
+    PackageReport,
+    GroupReport,
+    PackageProgress,
+    PackageHandler,
+    GroupHandler)
+
+
+MODULE = 'pulp_rpm.handlers.rpm'
+
+
+class TestPackageReport(TestCase):
+
+    @patch(MODULE + '.ContentReport')
+    def test_succeeded(self, content_report):
+        details = {
+            'resolved': [1, 2, 3],
+            'deps': [4, 5]
+        }
+
+        # test
+        report = PackageReport()
+        report.set_succeeded(details)
+
+        # validation
+        content_report.set_succeeded(details, 5)
+
+
+class TestGroupReport(TestCase):
+
+    @patch(MODULE + '.ContentReport')
+    def test_succeeded(self, content_report):
+        details = {
+            'resolved': [1, 2, 3],
+            'deps': [4, 5]
+        }
+
+        # test
+        report = GroupReport()
+        report.set_succeeded(details)
+
+        # validation
+        content_report.set_succeeded(details, 5)
+
+
+class TestPackageProgress(TestCase):
+
+    def test_init(self):
+        conduit = Mock()
+
+        # test
+        report = PackageProgress(conduit)
+
+        # validation
+        report.conduit = conduit
```

Comment:
Is there something missing? Should there be an assertion? Or will this assignment somehow fail if the test was not successful? It's not clear to me what success actually means here.